config/Dockerfiles/cloud-image-compose/virt-customize.sh

```diff
@@ -26,25 +26,14 @@ if [ $branch != "rawhide" ]; then
     branch=${branch:1}
 fi
 
-# Define proper install url
-if [[ $(curl -q https://dl.fedoraproject.org/pub/fedora/linux/development/ | grep "${branch}/") != "" ]]; then
-    INSTALL_URL="https://dl.fedoraproject.org/pub/fedora/linux/development/${branch}/Cloud/x86_64/images/"
-elif [[ $(curl -q https://dl.fedoraproject.org/pub/fedora/linux/releases/ | grep "${branch}/") != "" ]]; then
-    if [ "${branch}" -lt 28 ]; then
-        INSTALL_URL="https://dl.fedoraproject.org/pub/fedora/linux/releases/${branch}/CloudImages/x86_64/images/"
-    else
-        INSTALL_URL="https://dl.fedoraproject.org/pub/fedora/linux/releases/${branch}/Cloud/x86_64/images/"
-    fi
-else
-    echo "Could not find installation source! Exiting..."
-    exit 1
-fi
-
-# Temporary work around for broken rawhide image
-if [ $branch == "rawhide" ]; then
-    wget --quiet -r --no-parent -A 'Fedora-Cloud-Base-Rawhide*.qcow2' http://artifacts.ci.centos.org/artifacts/fedora-atomic/pipeline/rawhide/images/
-    DOWNLOADED_IMAGE_LOCATION=$(pwd)/$(find artifacts.ci.centos.org -name "*.qcow2" | head -1)
+if [ "${branch}" == "rawhide" ]; then
+    curl -L -k -O "https://jenkins-continuous-infra.apps.ci.centos.org/view/Fedora%20All%20Packages%20Pipeline/job/fedora-rawhide-image-test/lastSuccessfulBuild/artifact/Fedora-Rawhide.qcow2"
+    DOWNLOADED_IMAGE_LOCATION="$(pwd)/Fedora-Rawhide.qcow2"
+elif [ "${branch}" -eq 28 ]; then
+    curl -L -k -O "https://jenkins-continuous-infra.apps.ci.centos.org/view/Fedora%20All%20Packages%20Pipeline/job/fedora-f28-image-test/lastSuccessfulBuild/artifact/Fedora-28.qcow2"
```

Comment:
I think it'd be a little cleaner to not have the view in the url. You can remove view/Fedora%20.. (up to /job, but leave /job) and it will be the same link, but not dependent on the view name.

Reply:
good catch.